openpype/pipeline/context_tools.py

```diff
@@ -70,10 +88,12 @@ def install_host(host):
             avalon host-interface.
     """
     global _is_installed
+    global _modules_manager
```

Comment:
Line is not needed since `_get_modules_manager()` is called which already sets/reads the global variable.

Reply:
yep, fixed